src/styles/block/card.scss

```diff
@@ -0,0 +1,94 @@
+.card {
+  width: 200px;
+  border: 1px solid #e5e5e5;
+  border-radius: 5px;
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
```

Comment:
Organize your styles
I recommend you take a look at these styles code and use it in the feature 
[9elements](https://9elements.com/css-rule-order) or  [htmlAcademy](https://codeguide.academy/html-css.html#css-order)